dbt/models/default/schema/default.vw_pin_address.yml

```diff
@@ -1,6 +1,11 @@
 models:
   - name: default.vw_pin_address
     description: '{{ doc("view_vw_pin_address") }}'
+    config:
+      packages:
+        - "assesspy==1.1.*"
+        - "numpy==1.26.*"
+        - "pandas==2.*"
```

Comment:
**question (blocking):** Why are these here? Just for testing?

Reply:
Sorry about that, this was one of the things that prompted me to un-request your review after realizing I had requested it prematurely! I'll just DM you next time that happens instead of messing around with the GitHub review request settings. Fixed in [a5f5262](https://github.com/ccao-data/data-architecture/pull/435/commits/a5f5262cc49cff8db3659e31b1cab979588b44dc).